### DISCUSSION TITLE
Hover changing outside group properties

### DISCUSSION THREAD

**Discussion by lilvlad** [tailwindlabs/tailwindcss#11897] [Help] [answered]
Basically I have this "navbar" when active it has a blue pill, I want so on hover of the inactive pill, the active one blue background disappears and appears on the hovering one, but the "group" class only works with the elements inside the, how can I manipulate the outside elements on hover?
https://play.tailwindcss.com/e4eXTvtEr3

**Comment by wongjn** [comment]
Please consider proving your code as a [Tailwind Play](https://play.tailwindcss.com) – [images of code is not so helpful](https://meta.stackoverflow.com/a/285557/19655631).

**Reply by lilvlad** [reply]
Sorry my bad, I updated the question

**Reply by wongjn** [reply, marked as answer]
You could consider adding `group` on the `<ul>` wrapper, then adding `group-hover:bg-transparent` on the active pill. This works since when the _inactive_ pill is hovered, then `<ul>` is hovered too. However, this will mean that the active pill will have a transparent background color when you hover over it, so change `hover:bg-blue-700` to `group-hover:hover:bg-blue-700` to ensure this does not happen.

https://play.tailwindcss.com/oS45NhMBUe

**Reply by lilvlad** [reply]
Yes, worked perfectly, thank you so much.
Is there any way of just having one active pill and when hovered it would move the position hovered smoothly between each other, if you undesrtand me

**Reply by wongjn** [reply]
Yes, you'd want to look at using JavaScript, but that is out-of-scope of Tailwind.

**Reply by lilvlad** [reply]
I've managed to create an active pill in a div and play with the transform settings, but I can't manage to add a smooth transition to the "right" class, am I doing something wrong?
https://play.tailwindcss.com/8QMXTVZjJh
Update: I've managed to make it
https://play.tailwindcss.com/SeAoPjrT0I
Thanks for the help.
